test/BlogListing-test.js

```diff
@@ -15,16 +15,15 @@ describe('BlogListing', () => {
     expect(component.find('.blogListing')).to.have.length(1);
   });
 
-  it('contains an h2.blogListing-title', () => {
-    const title = component.find('h2');
+  it('contains an h1.blogListing-title', () => {
+    const title = component.find('h1');
     expect(title).to.have.length(1);
     expect(title.hasClass('blogListing-title')).to.equal(true);
     expect(title.text()).to.equal('title');
   });
 
   it('has a a link in the heading', () => {
-    const link = component.find('h2').find('a');
+    const link = component.find('.blogListing').find('a').at(1);
```

Comment:
@EdwinGuzman This is the next most logical test I can think of for `has a link in the heading` case.

Reply:
I see what's up. Check the `BlogListing` component. The link surrounds the h1 instead of what the test previously had which was the h2 surrounding the link. This check to see if the `.blogListing` class has a link, but it doesn't specify that it's a heading like the test description says. So how about getting the link `a` and then making sure that it has an `h1`. I think that'd be a better test.